tests/cnf/core/network/internal/define/nad.go

```diff
@@ -90,3 +91,39 @@ func createNadWithMasterPlugin(
 
 	return createdNad, nil
 }
+
+// CreateExternalNad creats an external network-attchment-definition using the br-ex interface.
+func CreateExternalNad(apiClient *clients.Settings, name, testNameSpace string) error {
+	glog.V(90).Info("Creating external BR-EX NetworkAttachmentDefinition")
+
+	// Define the master NAD plugin
+	macVlanPlugin, err := MasterNadPlugin(coreparams.OvnExternalBridge, "bridge", nad.IPAMStatic())
+	if err != nil {
+		glog.V(90).Infof("Failed to define master NAD plugin: %v", err)
+
+		return err
+	}
+
+	// Create the NetworkAttachmentDefinition
+	_, err = CreateNadWithMasterPlugin(apiClient, name, testNameSpace, macVlanPlugin)
+	if err != nil {
+		glog.V(90).Infof("Failed to create external NetworkAttachmentDefinition: %v", err)
+
+		return err
+	}
+
+	glog.V(90).Infof("Successfully created external NetworkAttachmentDefinition: %s", name)
+
+	return nil
+}
+
+// CreateNadWithMasterPlugin creates a MasterPlugin network attachment definition.
+func CreateNadWithMasterPlugin(
```

Comment:
It's duplication func, there is the fun  createNadWithMasterPlugin()

Reply:
removed